README.md

```diff
@@ -20,35 +20,24 @@ Create MDX files showcasing your code and Docz turns them into a live-reloading,
 
 ![docz example](https://cdn-std.dprcdn.net/files/acc_649651/S2YCID)
 
-## Table of Contents
-
-- [Why?](#why)
-- [Getting started](#getting-started)
-- [Build](#build)
-- [Deploy](#deploy)
-- [Examples](#examples)
-- [More info on docz.site](#more-info-on-doczsite)
-- [Used by](#used-by)
-- [Contributors](#contributors)
-- [Contributing](#contributing)
-
 ## Why?
 
-Documenting code is one of the most important and time-heavy processes when developing software.
+Documenting code is one of the most important and time-consuming tasks when developing software.
 
 A lot of time is spent on building and maintaining custom documentation sites.
 
-Docz enables you to quickly create live-reloading, seo-friendly, production-ready documentation sites with MDX and customize the look, feel and behavior when required by leveraging [GatsbyJS](https://www.gatsbyjs.org) and [Gatsby theme shadowing](https://www.gatsbyjs.org/docs/themes/shadowing/).
+Docz enables you to quickly create a live-reloading, SEO-friendly, production-ready documentation site with MDX and customize the look, feel and behavior when required by leveraging [GatsbyJS](https://www.gatsbyjs.org) and [Gatsby theme shadowing](https://www.gatsbyjs.org/docs/themes/shadowing/).
 
-## Getting started
+## About `create-docz-app`
```

Comment:
I wouldn't like to put it as this much emphasis in our README, because I wouldn't like to encourage other people to use that for this end, so please, could you change it for a heading-3 (aka `###`)?

Reply:
```suggestion
### About `create-docz-app`
```